measures/ResStockArguments/measure.rb

```diff
@@ -452,19 +458,30 @@ def run(model, runner, user_arguments)
     end
 
     # Plug Loads
-    args[:misc_plug_loads_television_annual_kwh] = 0.0 # "other" now accounts for television
-    args[:misc_plug_loads_television_usage_multiplier] = 0.0 # "other" now accounts for television
+    args[:misc_plug_loads_television_usage_multiplier] = Float(args[:misc_plug_loads_television_usage_multiplier].to_s) * args[:misc_plug_loads_television_2_usage_multiplier]
     args[:misc_plug_loads_other_usage_multiplier] = Float(args[:misc_plug_loads_other_usage_multiplier].to_s) * args[:misc_plug_loads_other_2_usage_multiplier]
     args[:misc_plug_loads_well_pump_usage_multiplier] = Float(args[:misc_plug_loads_well_pump_usage_multiplier].to_s) * args[:misc_plug_loads_well_pump_2_usage_multiplier]
     args[:misc_plug_loads_vehicle_usage_multiplier] = Float(args[:misc_plug_loads_vehicle_usage_multiplier].to_s) * args[:misc_plug_loads_vehicle_2_usage_multiplier]
 
+    # Television
+    if args[:misc_plug_loads_television_annual_kwh].to_s == Constants.Auto
+      if [HPXML::ResidentialTypeSFD].include?(args[:geometry_unit_type])
+        args[:misc_plug_loads_television_annual_kwh] = 309.9 + 96.11 * args[:geometry_unit_num_occupants] + 0.06 * args[:geometry_unit_cfa] # RECS 2020
+      elsif [HPXML::ResidentialTypeSFA].include?(args[:geometry_unit_type])
+        args[:misc_plug_loads_television_annual_kwh] = 272.66 + 74.16 * args[:geometry_unit_num_occupants] + 0.08 * args[:geometry_unit_cfa] # RECS 2020
+      elsif [HPXML::ResidentialTypeApartment].include?(args[:geometry_unit_type])
+        args[:misc_plug_loads_television_annual_kwh] = 171.97 + 76.09 * args[:geometry_unit_num_occupants] + 0.11 * args[:geometry_unit_cfa] # RECS 2020
+      end
```

Comment:
CFA may have a weak correlation here. Consider replacing with some combination of:
* no replacement (number of occupants only)
* number of bedrooms
* number of TVs

(OS-HPXML "TV other" default is calculated based on number of bedrooms.)

Reply:
Going with number of occupants and number of TVs.